runtime/account_test.go

```diff
@@ -507,19 +515,42 @@ func TestRuntimeAuthAccountKeysAdd(t *testing.T) {
 	)
 
 	require.NoError(t, err)
-	assert.Len(t, storage.keys, 1)
+	assert.Len(t, storage.keys, 2)
 
-	require.Len(t, storage.events, 2)
+	require.Len(t, storage.events, 3)
 
 	assert.EqualValues(t,
 		stdlib.AccountCreatedEventType.ID(),
 		storage.events[0].Type().ID(),
 	)
 
+	key0AddedEvent := storage.events[1]
+	key1AddedEvent := storage.events[2]
+
 	assert.EqualValues(t,
 		stdlib.AccountKeyAddedFromPublicKeyEventType.ID(),
-		storage.events[1].Type().ID(),
+		key0AddedEvent.Type().ID(),
 	)
+
```

Comment:
While at it, could you maybe please also add assertions for the first and second field?